### ISSUE TITLE
sra_toolkit 2.6.2 cannot be installed on Main

### ISSUE THREAD

**Issue by martenson** [galaxyproject/tools-iuc#837]
The new version(Galaxy version 1.3.0) has trouble installing its dependency sra_toolkit	2.6.2

```
Configuring SRA-TOOLS package
checking system type... Linux
checking OS distributor...
checking machine architecture... x86_64
checking SRA-TOOLS version... 2.6.2
checking for supported architecture... x86_64 (64 bits) is supported
checking for supported OS... Linux (linux) is supported
checking for supported tool chain... gcc tool chain is supported
checking for g++... yes
checking whether gcc accepts -Wno-array-bounds... yes
checking whether g++ accepts -static-libstdc++... yes
checking for fuse library... no
checking for hdf5 library... no
checking for magic library... no
checking for xml2 library... no
checking for ngs-sdk package...
includes... /cvmfs/main.galaxyproject.org/deps/ngs_sdk/1.2.3/iuc/package_ngs_sdk_1_2_3/3da0f4d34e92
libraries... /cvmfs/main.galaxyproject.org/deps/ngs_sdk/1.2.3/iuc/package_ngs_sdk_1_2_3/3da0f4d34e92/lib64
includes: /cvmfs/main.galaxyproject.org/deps/ngs_sdk/1.2.3/iuc/package_ngs_sdk_1_2_3/3da0f4d34e92/include
libraries: /cvmfs/main.galaxyproject.org/deps/ngs_sdk/1.2.3/iuc/package_ngs_sdk_1_2_3/3da0f4d34e92/lib64
checking for ncbi-vdb package source files and build results...
includes... no
includes... no
libraries... /cvmfs/main.galaxyproject.org/deps/ncbi_vdb/2.6.2/iuc/package_ncbi_vdb_2_6_2/be38d36c735d/lib64
no
libraries... /cvmfs/main.galaxyproject.org/deps/ncbi_vdb/2.6.2/iuc/package_ncbi_vdb_2_6_2/be38d36c735d/ncbi-vdb/linux/gcc/x86_64/rel/ilib
configure: error: required ncbi-vdb package not found.
```

**Comment by mvdbeek** [comment]
I'll have a look.

**Comment by martenson** [comment]
note: the previous version installed successfully

**Comment by mvdbeek** [comment]
I just installed this on our server and all went fine. Can you post the beginning of the INSTALLATION.log?
Ours is here, for comparison:
https://gist.github.com/mvdbeek/4bf1f28b928fc6fd0bd8a9a8f48cbda3

**Comment by martenson** [comment]
https://gist.github.com/martenson/91cba0c99a5288dce43fda07cc710723

**Comment by mvdbeek** [comment]
Hmm, I assume the rest of the dependencies is green, right?
What's the content of /cvmfs/main.galaxyproject.org/deps/ncbi_vdb/2.6.2/iuc/package_ncbi_vdb_2_6_2/be38d36c735d/ncbi-vdb/linux/gcc/x86_64/rel/ilib ?
What OS is main running?

**Comment by martenson** [comment]
@mvdbeek :
```
libalign-access.a  libkapp.a         libklib.a        libksrch.a         libncbi-bam.a   libsraxf.a      libwaxf.a        libwvxf.a
libalign-reader.a  libkapp-norsrc.a  libkns.a         libktst.a          libngs.a        libtui.a        libwgsxf.a       libwwgsxf.a
libalign-writer.a  libkdb.a          libkproc.a       libkxfs.a          libngs-jni.a    libtui_cpp.a    libwkdb.a        libz.a
libascp.a          libkfc.a          libkq.a          libkxml.a          libngs-py.a     libvdb.a        libwsradb.a
libaxf.a           libkfg.a          libkrypto.a      libload.a          libsradb.a      libvdb-blast.a  libwsraschema.a
libbz2.a           libkfs.a          libkryptotest.a  libloader.a        libsrareader.a  libvfs.a        libwsraxf.a
libjudy.a          libkfs-nommap.a   libksproc.a      libmagic-static.a  libsraschema.a  libvxf.a        libwvdb.a
```

OS: `CentOS Linux release 7.2.1511 (Core)`

**Comment by martenson** [comment]
edited the comment above

**Comment by mvdbeek** [comment]
Hmm, this all looks normal ... I'll see if I can compile this on a centos docker image. 
Any chance you could get this through conda?

**Comment by martenson** [comment]
Can't do Conda on Main yet I think :/

**Comment by mvdbeek** [comment]
ughh, this is a pain ... so it installs fine by hand on a centos:6.7 docker image, but I couldn't quite figure out how to get python2.7 on centos to test the galaxy package. One attempt was to create a new package for 2.7.0, but the installer is super-intrusive and needs sudo to write to /etc/, so we have to patch the makefile ... ugh

**Comment by martenson** [comment]
@mvdbeek let's wait for @natefoo maybe he will have some ideas

**Comment by natefoo** [comment]
I fixed this for Main's 2.6.2 install manually, see the details in this comment: https://github.com/galaxyproject/galaxy/issues/2533#issuecomment-249963999

For subsequent versions we are using conda dependencies, which are working fine.
